pkg/pxc/service.go

```diff
@@ -144,6 +145,13 @@ func NewServiceProxySQLUnready(cr *api.PerconaXtraDBCluster) *corev1.Service {
 	return obj
 }
 
+func ProxySQLUnreadyServiceNamespacedName(cr *api.PerconaXtraDBCluster) types.NamespacedName {
+	return types.NamespacedName{
+		Name:      cr.Name + "-proxysql-unready",
+		Namespace: cr.Namespace,
+	}
+}
+
```

Comment:
I would suggest to move all such funcs into `api` package

Reply:
it's a part of service, not our API

Reply:
It's also a part of other packages.
https://github.com/percona/percona-xtradb-cluster-operator/blob/K8SPXC-609/pkg/pxc/app/statefulset/proxysql.go#L47
This is why I want to put it in `api` package (or any other new package) which can be used from other packages to make code more consistent.

Reply:
`+ "-proxysql-unready"` used in a couple of other places. As well as `+ "-proxysql"` etc.
Let's either do abstractions that work everywhere or avoid them at all (` metav1.ObjectMeta{Name: cr.Name + "-proxysql-unready"}` etc)

Reply:
f474c880

Reply:
@dAdAbird 
We already moved this functions to api package.
Is it's ok?